packages/dappmanager/src/modules/stakerConfig/params.ts

```diff
@@ -0,0 +1,36 @@
+import { Network } from "@dappnode/common";
+import {
+  ExecutionClientOrSignerVersion,
+  LodestarStakersMinimumVersions
+} from "./types";
+
+const lodestarStakersPraterMinimumVersions: ExecutionClientOrSignerVersion<"prater"> =
+  {
+    "goerli-nethermind.dnp.dappnode.eth": "1.0.8",
+    "goerli-geth.dnp.dappnode.eth": "0.4.27",
+    "goerli-erigon.dnp.dappnode.eth": "0.1.4",
+    "goerli-besu.dnp.dappnode.eth": "0.1.1",
+    "web3signer-prater.dnp.dappnode.eth": "0.1.17"
```

Comment:
`0.1.17` does not work with Lodestar as brain does send the correct API, need https://github.com/dappnode/StakingBrain/pull/196 to be merged, released and web3signer to be updated